src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -177,14 +177,27 @@ const SystemsExposedTable = (props) => {
         );
     };
 
+    const getSelectedSystemsWithName = () => {
```

Comment:
it would be better to use different function name cause in case you select more than one system you don't return the systems with the name but just the id.